lib/turnip/placeholder.rb

```diff
@@ -4,6 +4,7 @@ class Match < Struct.new(:regexp, :block);end
 
     class << self
       def add(name, &block)
+        warn "Placeholder :#{name} was replaced at #{caller_locations.first}." if placeholders.key?(name)
```

Comment:
Just realised this doesn't give the correct `calller_location` when called with the dsl.